src/pages/signin/LoginForm/index.js

```diff
@@ -1,19 +0,0 @@
-import React from 'react';
-import LoginFormNarrow from './LoginFormNarrow';
-import LoginFormWide from './LoginFormWide';
```

Comment:
If we are not using this anymore then we can also delete the `LoginForm/LoginFormWide.js`?

Reply:
Hmm, it was added back in a merge.. I'll delete it